Triangulation_2/doc/Triangulation_2/CGAL/Triangulation_2_projection_traits_3.h

```diff
@@ -0,0 +1,79 @@
+namespace CGAL{
+
+/*!
+ \ingroup PkgTriangulation2TriangulationClasses
+
+  The class `Triangulation_2_projection_traits_3` is an extension of the `Projection_traits_xy_3`,
+  `Projection_traits_xz_3` and `Projection_traits_yz_3` traits classes,that allows the use
+  of 2D algorithms on the projections of 3D data on any plane.
+
+  \cgalHeading{Parameters}
+  The template parameter `K` has to
+  be instantiated by a model of the `K` concept.
+  `Triangulation_2_projection_traits_3` uses types
+  and predicates defined in `K`.
```

Comment:
```suggestion
  \tparam K must be a model of `Kernel`.
```

Reply:
I guess this is copied from the other traits class, so it could be done for the others too.

Reply:
Related to what @sgiraudot what talking about about "what type of Kernel is expected from a class", it seems needed here to address what is provided by this class depending on `K` since constructions are used to project points. Something like:

> \tparam K must be a model of `Kernel`. 
>
> \note Internal constructions (projections) are used in the predicate and construction functors of this class. If `K` is a model of `Kernel` providing exact constructions or if `K` is a `CGAL::Filtered_kernel`, which is detected when `K::Has_filtered_predicates` exists and is `true` (such as for `CGAL::Exact_predicates_inexact_constructions_kernel`), this class automatically provides exact predicates.

? It is not the easiest to follow...